caffe2/CMakeLists.txt

```diff
@@ -325,6 +325,10 @@ set(GENERATED_CXX_TORCH_CUDA
   "${TORCH_SRC_DIR}/csrc/inductor/aoti_torch/generated/c_shim_cuda.cpp"
   )
 
+set(GENERATED_CXX_TORCH_XPU
+  "${TORCH_SRC_DIR}/csrc/inductor/aoti_torch/generated/c_shim_xpu.cpp"
```

Comment:
Missing `extend`?

Reply:
NVM: after reading a few more lines, I realized you actually have two parts of c_shim_xpu,

Reply:
@desertfire yes, as I descripted in the PR description above, there is 
```
torch/include/torch/csrc/inductor/aoti_torch/generated/c_shim_xpu.h
torch/include/torch/csrc/inductor/aoti_torch/generated/c_shim_xpu.cpp
torch/include/torch/csrc/inductor/aoti_torch/generated/extend/c_shim_xpu.h
torch/include/torch/csrc/inductor/aoti_torch/generated/extend/c_shim_xpu.cpp
```
The extend part is generated from out-of-tree torch-xpu-ops, which call `python -m torchgen.gen --source-path third_party/torch-xpu-ops/yaml/ --xpu --aoti-extend --update-aoti-c-shim --aoti-insall-dir=torch/include/torch/csrc/inductor/aoti_torch/generated/extend/`, and the generated `extend/c_shim_xpu.cpp` is built  into libtorch_xpu_ops.a. So  `extend/c_shim_xpu.cpp`  is added in cmake file of torch-xpu-ops, not in pytorch cmake file.